### PR TITLE
wire: Add tests for BlockHeader (From)Bytes.

### PR DESCRIPTION
**Pull request by jozn**
This increase test coverage of BlockHeader.

In following days, I will increase the test coverage of wire. Some test suites need to be refactored to allow for error paths and wrong inputs, I will keep the test cases data but reorganize them and make them more structured this will allow for increase test coverage and easier understanding and covers more samples.
I will keep the PR short and separate for each Message.
After this, I will work with other packages and increase their test coverage and quality.

### CODE REVIEW

File: wire/blockheader_test.go [hunk 1]
```diff
@@ -244,6 +244,33 @@ func TestBlockHeaderWire(t *testing.T) {
 				spew.Sdump(&bh), spew.Sdump(test.out))
 			continue
 		}
+
+		//test BlockHeader Bytes()
```
Comment (davecgh) [inline wire/blockheader_test.go]: Please match the style used by the comments throughout the rest of the tests.

File: wire/blockheader_test.go [hunk 2]
```diff
@@ -244,6 +244,33 @@ func TestBlockHeaderWire(t *testing.T) {
...
+			continue
+		}
+
+		if !bytes.Equal(bts, test.buf) {
+			t.Errorf("Bytes #%d\n got: %s want: %s", i,
+				spew.Sdump(&bts), spew.Sdump(test.out))
+			continue
+		}
+
+		//test BlockHeader FromBytes()
```
Comment (davecgh) [inline wire/blockheader_test.go]: Same here.

**Comment by jozn** [comment]
Thanks Dave, I'll change the PR as requested and improve it.

**Comment by davecgh** [comment]
Please let me know once this is updated.

**Comment by jozn** [comment]
Updated. Sorry I didn't see your comment in here.
At first, I wanted to improve this PR tests, for now I want to move to other packages and simultaneously add tests for other committed filters of wire.

**Comment by jozn** [comment]
This PR is finalized.
After submitting it before I double check the code comments again and improve it further and now the comments match the style of other comments in the test suite.

**Pull request merged**